build.pl

```diff
@@ -248,15 +248,14 @@ ()
    System( "find", $GLOBAL_BUILD_DIR, "-type", "f", "-name", ".built.*", "-delete" ) if ( $ENV{ENV_CACHE_CLEAR_FLAG} );
 
    my @TP_JARS = (
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/ant-1.7.0-ziputil-patched.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/ant-contrib-1.0b1.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/ews_2010-1.0.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/jruby-complete-1.6.3.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/plugin.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/servlet-api-3.1.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/unboundid-ldapsdk-2.3.5-se.jar",
-      "http://$GLOBAL_BUILD_THIRDPARTY_SERVER/ZimbraThirdParty/third-party-jars/zimbrastore-test-1.0.jar",
-   );
+      "https://files.zimbra.com/repository/ant-1.7.0-ziputil-patched/ant-1.7.0-ziputil-patched-1.0.jar",
```

Comment:
We need to remove static references gradually, filed tracking tasks:

https://jira.corp.synacor.com/browse/ZCS-1185 (Generate ant-1.7.0-ziputil-patched-1.0.jar artifact and remove static reference of file from files.zimbra.com in build.pl)

https://jira.corp.synacor.com/browse/ZCS-1186 (Generate ant-contrib-1.0b1.jar artifact and remove static reference of file from files.zimbra.com in build.pl)

https://jira.corp.synacor.com/browse/ZCS-1187 (Generate ews_2010-1.0.jar artifact and remove static reference of file from files.zimbra.com in build.pl)

https://jira.corp.synacor.com/browse/ZCS-1188 (Generate jruby-complete-1.6.3.jar artifact and remove static reference of file from files.zimbra.com in build.pl)

https://jira.corp.synacor.com/browse/ZCS-1189 (Generate plugin.jar artifact and remove static reference of file from files.zimbra.com in build.pl)

https://jira.corp.synacor.com/browse/ZCS-1190 (Generate servlet-api-3.1.jar artifact and remove static reference of file from files.zimbra.com in build.pl)